cni_stop.sh

```diff
@@ -113,8 +113,26 @@ function unload_module {
 
 let status=0
 
+function load_inbox_drivers {
+    modprobe mlx5_core
+    modprobe ib_core
+}
+
+function delete_nic_operator_namespace {
+    if [[ !  -f "$WORKSPACE/mellanox-network-operator/deploy/operator-ns.yaml" ]];then
+        return 0
+    fi
+    nic_operator_namespace_file=$WORKSPACE/mellanox-network-operator/deploy/operator-ns.yaml
+    operator_namespace=$(grep -E '^[ ]+name:' $nic_operator_namespace_file | head -n 1 | cut -d: -f2 | tr -d ' ')
+    kubectl delete namespace $operator_namespace
```

Comment:
you can just: `kubectl delete -f $nic_operator_namespace_file`

Reply:
ah yes, my mistake. will change that.